src/plugins/pgcomparator_cdc_operator.py

```diff
@@ -39,6 +39,8 @@ def __init__(
         key_column: Optional[str] = None,
         use_key: bool = False,
         db_conn_id: Optional[str] = None,
+        no_deletes: bool = False,
+        database_password: str = "DSO_DB_POSTGRES_PASSWORD",
```

Comment:
Deze naam suggereert dat dit het wachtwoord is. Wellicht iets als `db_passwd_env_var`?

Als je de env var on-the-fly aan `subprocess.run` mee zou kunnen geven, zou je deze variabele helemaal niet nodig hebben, maar ik weet niet of dat lukt.

Reply:
Ik kreeg em met geen mogelijkheid aan de praat. Er wordt on the fly niet toegestaan op een export envar te doen. Envvars die reeds bestaan (via de docker-compose) zijn wel ok